docs/docs/rule.md

```diff
@@ -35,6 +35,7 @@ You can define a rule using two styles
 | actions | `[]` | array[[RuleAction](#ruleaction)] | Executes an action after bot's message
 | preActions | `[]` | array[[RuleAction](#ruleaction)] | Executes an action before bot's message
 | postActions | `[]` | array[[RuleAction](#ruleaction)] | Executes an action after user's answer
+| skip | - | RuleTypeSkip | Skip rule based on conditional function
```

Comment:
1. change default to `false`
1. in type, put `boolean | (output, rule, bot) => boolean`